com.woltlab.wcf/package.xml

```diff
@@ -102,5 +102,8 @@
 	<instructions type="update" fromversion="3.1.15">
 		<instruction type="file">files_update.tar</instruction>
 		<instruction type="template">templates_update.tar</instruction>
+		<!-- Possibly add an update script that deletes orphaned embedded objects.
+			see: https://github.com/WoltLab/WCF/issues/3429
+		-->
```

Comment:
Can you please provide the script with the PR? I will stumble across this comment weeks later when working on the next update and then have to dive into this topic to see what actually needs to be done.

Reply:
Sure, I'll work on this.

Reply:
Okay, I added the script. I've tested it, but please have a careful look at it during review.